web/impact/impact/tests/test_user_list_view.py

```diff
@@ -108,16 +114,257 @@ def test_get(self):
             assert user2.email in emails
             assert user_count == response.data["count"]
 
-    def test_get_with_limit(self):
-        UserContext().user
-        UserContext().user
+    def test_get_returns_correct_count_attribute(self):
+        for _ in range(10):
+            UserContext()
         with self.login(email=self.basic_user().email):
-            url = self.url + "?limit=1"
             user_count = User.objects.count()
+            response = self.client.get(self.url)
+            assert user_count == response.data["count"]
```

Comment:
Are you going with `assert` instead of `self.assertEqual` and the like because the file already uses that style exclusively? I prefer the latter style because the failure messages are more informative. But I think I've seen you post similar review comments in the past so perhaps there's a good reason! Not a blocker.

Reply:
for consistency's sake. plus my hope now is that we'll change to pytest, which produces informative errors with regular assert statements.